### PR TITLE
Refactoring for routing screen

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review

File: go/base/static/js/test/runner.html [hunk 1]
```diff
@@ -77,25 +77,11 @@
...
-    <script src="../src/campaign/dialogue/dialogue.js"></script>
-    <script src="../src/campaign/dialogue/models.js"></script>
-    <script src="../src/campaign/dialogue/connections.js"></script>
-    <script src="../src/campaign/dialogue/states/states.js"></script>
-    <script src="../src/campaign/dialogue/states/partials.js"></script>
-    <script src="../src/campaign/dialogue/states/dummy.js"></script>
-    <script src="../src/campaign/dialogue/states/choice.js"></script>
-    <script src="../src/campaign/dialogue/states/freetext.js"></script>
-    <script src="../src/campaign/dialogue/states/end.js"></script>
-    <script src="../src/campaign/dialogue/diagram.js"></script>
```
Comment (hodgestar) [inline go/base/static/js/test/runner.html:94]: Can we really remove all these things from the test runner?

Comment (justinvdm) [inline go/base/static/js/test/runner.html:94]: Ya, these were left behind from a while ago, and shouldn't be here in the first place :) I think I forgot to remove them when I changed the directory structure for the client side.

Comment (hodgestar) [inline go/base/static/js/test/runner.html:94]: Woot.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**